roles/redpanda_broker/tasks/install-rp-deb.yml

```diff
@@ -12,15 +17,28 @@
   environment:
     https_proxy: "{{ https_proxy_value | default('') }}"
     http_proxy: "{{ https_proxy_value | default('') }}"
-  when: https_proxy_value is defined and https_proxy_value | length > 0
+  when:
+    - https_proxy_value is defined
+    - https_proxy_value | length > 0
+    - (using_gcp | default(false)) is false
 
-- name: Install redpanda from repository
+- name: Install redpanda from repository (Standard)
   ansible.builtin.package:
     name:
       - "{{ redpanda_package_name }}"
     state: "{{ redpanda_install_status }}"
     update_cache: true
-  when: https_proxy_value is not defined or https_proxy_value | length == 0
+  when: https_proxy_value is not defined and (using_gcp | default(false)) is false
+
+- name: Install redpanda from repository (GCP)
+  ansible.builtin.package:
+    name:
+      - "{{ redpanda_package_name }}"
+    state: "{{ redpanda_install_status }}"
+    update_cache: true
+    allow_unauthenticated: yes
+  when: (using_gcp | default(false)) is not false
```

Comment:
[nit] So the difference between Standard and GCP is the `allow_unauthenticated` ? in that case, can't we leave just one with:
```
 allow_unauthenticated: (using_gcp | default(false))
 ```
 ?